packages/node-bridge/launcher.js

```diff
@@ -132,6 +142,54 @@ function getVercelLauncher({
   };
 }
 
+/**
+ * @param {Function} listenerHandler
+ * @returns {(req: import('./types').VercelRequest, res: import('./types').VercelResponse) => void}
+ */
+function wrapListener(listenerHandler) {
+  // TODO: how can we best test the timeout and warning behavior?
```

Comment:
We could pass the timeout value through an env var to `vc dev` from an integration test